kratos.gid/apps/CDEM/write/writeMDPA_Walls.tcl

```diff
@@ -299,31 +300,31 @@ proc CDEM::write::writeWallConditionMeshes { } {
             write::WriteString "    IS_GHOST $is_ghost"
             write::WriteString "    IDENTIFIER [write::transformGroupName $group]"
 
-            set material_analysis [write::getValue DEMTestMaterial Active]
-            if {$material_analysis == "true"} {
-                set is_material_test [write::getValueByNode [$group_node selectNodes "./value\[@n='MaterialTest'\]"]]
-                if {$is_material_test == "true"} {
-                    set as_condition [write::getValueByNode [$group_node selectNodes "./value\[@n='DefineTopBot'\]"]]
-                    if {$as_condition eq "top"} {
-                        write::WriteString "    TOP 1"
-                        write::WriteString "    BOTTOM 0"
-                    } else {
-                        write::WriteString "    TOP 0"
-                        write::WriteString "    BOTTOM 1"
-                    }
-                }
-            } else {
-                    write::WriteString "    TOP 0"
-                    write::WriteString "    BOTTOM 0"
-            }
-
-            set GraphPrint [write::getValueByNode [$group_node selectNodes "./value\[@n='GraphPrint'\]"]]
-            if {$GraphPrint == "true" || $material_analysis == "true"} {
-                set GraphPrintval 1
-            } else {
-                set GraphPrintval 0
-            }
-            write::WriteString "    FORCE_INTEGRATION_GROUP $GraphPrintval"
+            # set material_analysis [write::getValue DEMTestMaterial Active]
```

Comment:
Why are we commenting out all this section? Isn't it needed anymore? We want to run tests in the future.
Anyways, if removal is correct we just just delete the lines.